### DISCUSSION TITLE
Library Strategy - CDE 6273393

### DISCUSSION THREAD

**Discussion by GeoffLyle** [CBIIT/ccdi-federation-api#48] [Metadata Field Ratification]
caDSR Link: [6273393](https://cadsr.cancer.gov/onedata/dmdirect/NIH/NCI/CO/CDEDD?filter=CDEDD.ITEM_ID=6273393%20and%20ver_nr=1)

Definition: "The overall strategy for the collection of double stranded DNA fragments flanked by oligonucleotide sequence adapters to enable their analysis by high-throughput sequencing."

Issues with CDE:
Issue with orthogonal concepts in the same CDE (“WGS”, “WXS”, “RNA-Seq”, “Validation”).  Likely "Validation" should be removed from the CDE.

Proposed Solutions:

- Remove “Validation” as a permissible value.  Create a version 2.0 without this PV.​
- Subset the CDE.

**Comment by GeoffLyle** [comment]
Slack Discussion:

_From Clay McLeod on October 25th, 2023_ - For [CDE 6273393](https://cadsr.cancer.gov/onedata/dmdirect/NIH/NCI/CO/CDEDD?filter=CDEDD.ITEM_ID=6273393%20and%20ver_nr=1) ("Sequencing Library Technique Type"), I unfortunately again find the same thing: some permissible values capture what is traditionally referred to as the "strategy" of the sequencing (e.g., "WGS", "WXS", "RNA-Seq", etc) while some permissible values describe the intent of the strategy (e.g., "Validation" described as "The act of validating; finding or testing the truth of something."—in theory, many of the sequencing strategies listed int he permissible values could be used to validate a finding).

_From Patrick Dunn on November 27th, 2023_ - Discussed problem of having "Validation" as a permissible value.
 caDSR agrees "Validation" should be removed.

_Geoff Lyle's data harmonization subgroup meeting notes on November 17th, 2023_
Discussed problem of having "Validation" as a permissible value.
Proposed Solutions:

- Create a new version of the Library Strategy CDE without "Validation" as a permissible value.
- Subset the CDE permissible values (remove "Validation") for the Federation harmonized field.

**Comment by jknable** [comment]
“Validation” as a permissible value as removed on 1/29/24.

**Comment by lydiabrogers** [comment]
Harmonization team approved 2/16
